charts/stable/nextcloud/templates/_configmap.tpl

```diff
@@ -207,6 +209,17 @@ nginx-config:
           listen {{ .Values.service.main.ports.main.port }};
           absolute_redirect off;
 
+          # Forward Notify_Push "High Performance Backend"  to it's own container
+          location ^~ /push/ {
+              # The trailing "/" is important!
+              proxy_pass http://{{ printf "%v-notify" $fullname }}:{{ .Values.service.notify.ports.notify.targetPort }}/;
+              proxy_http_version 1.1;
+              proxy_set_header Upgrade $http_upgrade;
+              proxy_set_header Connection "Upgrade";
+              proxy_set_header Host $host;
+              proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+          }
```

Comment:
Yes this is fine for now. 
Do you mind adding a comment with a TODO and why this is not moved to ingress completely yet?

Reply:
Can't remember the why right now. 
I've added it for now/testing. 
I'll add the todo to not lost track o fit